### ISSUE TITLE
Bootstrapping database from scratch is not working

### ISSUE THREAD

**Issue by obscurerichard** [freezingsaddles/freezing-model#4]
There are multiple roadblocks in the way of bootstrapping an environment from scratch, with an empty databse. It may fail saying it can't find a table.  I tried for a couple weeks to pick away at this without success and eventually I loaded a dump of the production database in my local dev environment to work around it. 

It's entirely possible that I just missed an essential step that should be obvious and we need to get the docs improved.

The reading I did on alembic seems to indicate it can do most of the database bootstrapping by itself, see for example https://alembic.sqlalchemy.org/en/latest/cookbook.html  - I kind of expected there to be a blank schema migration as the first thing, but it's not there.

TODO: put in console trace of where this is now

**Comment by hozn** [comment]
The issue here was a bug in the logic that was calculating which tables to register as "managed".  I think this table registration system was a bit over-engineered -- and the fact that you had to import the `freezing.model.orm` module before you could call `init_model` was really not-great.  So I reworked this to make the enumeration of mapped tables explicit.

@obscurerichard , it works fine on my machine creating db from scratch.   I just bumped the patch-level version, since hopefully nothing else about the model is changed.  I did *not* update the dependency in freezing-web, etc.  Though I did tag 0.3.3.

**Comment by obscurerichard** [comment]
I tested this on my local dev environment by doing the following. In MySQL:

```
drop database freezing;
create database freezing;
grant all on freezing.* to freezing;
```
Then I started up `freezing-web` and it bootstrapped from scratch just like it was supposed to.

I also made sure that the new view went in in #11 and I bumped the tag to 0.3.4 to match here and in `freezingsaddles/freezing-web`.

Thank you so much @hozn for tying a bow on this!
